src/controllers/userController.ts

```diff
@@ -0,0 +1,45 @@
+import * as HttpStatus from 'http-status-codes';
+import * as Koa from 'koa';
+import * as passport from 'koa-passport';
+import { IUser } from '../interfaces/';
+import { confirmUserEmail, registerUser } from '../services/';
+import { getToken } from '../utils';
+
+export const signUp = async (ctx: Koa.Context) => {
+  if (!ctx) {
+    return;
+  }
+  try {
+    await registerUser(ctx.request.body);
+    ctx.status = HttpStatus.CREATED;
+    ctx.body = 'OK';
```

Comment:
Why do you need to hardcode the text here? 201 is just enough.
Plus this brings in problems with i18n.